consensus/consensus.go

```diff
@@ -100,6 +100,8 @@ type Engine interface {
 	//
 	// Note, the method returns immediately and will send the result async. More
 	// than one result may also be returned depending on the consensus algorithm.
+	// Stop is triggered by closing the @stop channel hence, any receive from
+	// @stop will succeed without blocking.
```

Comment:
This newly added comment is not particularly helpful. The only operation you can do with this stop channel is to read from it, so there's not much reason to explicitly lay out that you should read from it and it means stop.

Reply:
Not really. I was doing a private updated, and I had to verify if the channel is closed or fed with information. There is a significant difference: closed channels will always respond with a zero value. None-closed channels will respond exactly once per data  inserted into the channel.

Reply:
If the comment is not clear then I'm happy to edit it though.

Reply:
Please don't use `@` notation anywhere

Reply:
I've removed it few days ago (the snippet above is market as outdated).

Reply:
![Screenshot from 2020-05-20 10-36-32](https://user-images.githubusercontent.com/129561/82418659-f03bcc00-9a85-11ea-8b38-c7f9ea0ac69e.png)

Reply:
Ah, apologise. I was looking at different file (`clique.go`). I've just updated the comment.